include/zenoh/api/reply.hxx

```diff
@@ -84,7 +84,16 @@ class Reply : public Owned<::z_owned_reply_t> {
         return {};
     }
 #endif
+
+    /// @brief Construct a shallow copy of this Reply.
+    ///
+    /// The reply will be sent only when the last clone is destroyed.
+    Reply clone() const {
+        Reply q(zenoh::detail::null_object);
```

Comment:
Could you please also rename this "q" variable to something meaningful? E.g. `Reply reply` or something like that